source/extensions/filters/http/gzip/gzip_filter.cc

```diff
@@ -28,10 +28,11 @@ GzipFilterConfig::GzipFilterConfig(const envoy::extensions::filters::http::gzip:
       compression_level_(compressionLevelEnum(gzip.compression_level())),
       compression_strategy_(compressionStrategyEnum(gzip.compression_strategy())),
       memory_level_(memoryLevelUint(gzip.memory_level().value())),
-      window_bits_(windowBitsUint(gzip.window_bits().value())) {}
+      window_bits_(windowBitsUint(gzip.window_bits().value())),
+      chunk_size_(gzip.chunk_size().value() > 0 ? gzip.chunk_size().value() : 4096) {}
```

Comment:
Great find! Let's have a ref/discussion/comment here how others set this value?

Reply:
Sure -- let me add something

Reply:
Hmm, maybe the proto file is a better place for this discussion/comment?

Reply:
👍